composer.lock

```diff
@@ -11000,54 +11004,6 @@
                 "source": "https://git.drupalcode.org/project/views_tree"
             }
         },
-        {
-            "name": "drupal/virtual_tour",
-            "version": "2.0.0-alpha1",
-            "source": {
-                "type": "git",
-                "url": "https://git.drupalcode.org/project/virtual_tour.git",
-                "reference": "2.0.0-alpha1"
-            },
-            "dist": {
-                "type": "zip",
-                "url": "https://ftp.drupal.org/files/projects/virtual_tour-2.0.0-alpha1.zip",
-                "reference": "2.0.0-alpha1",
-                "shasum": "d6efd9604a356561df008d9881ade5c13ff6f3a1"
-            },
-            "require": {
-                "drupal/core": "^8 || ^9"
-            },
-            "type": "drupal-module",
-            "extra": {
-                "drupal": {
-                    "version": "2.0.0-alpha1",
-                    "datestamp": "1588250556",
-                    "security-coverage": {
-                        "status": "not-covered",
-                        "message": "Project has not opted into security advisory coverage!"
-                    }
-                }
-            },
-            "notification-url": "https://packages.drupal.org/8/downloads",
-            "license": [
-                "GPL-2.0-or-later"
-            ],
-            "authors": [
-                {
-                    "name": "rahulrasgon",
-                    "homepage": "https://www.drupal.org/user/3513914"
-                },
-                {
-                    "name": "sd9121",
-                    "homepage": "https://www.drupal.org/user/3513915"
-                }
-            ],
-            "description": "Provides an field formatter to Virtual Tour effect on an image.",
-            "homepage": "https://www.drupal.org/project/virtual_tour",
-            "support": {
-                "source": "https://git.drupalcode.org/project/virtual_tour"
-            }
-        },
```

Comment:
This is removing the `virtual_tour` module, which I'm assuming was unintentional, @joewhitsitt?

Reply:
Or maybe not, since I'm not seeing it in the `composer.json` file.

Reply:
So, now I'm remembering that we didn't go with `virtual_tour` in favor of doing our implementation, so this is fixing that, I guess.

Reply:
>got rid of a virtual_tour remnant.

Yeah, this is still in main and didn't get cleaned up properly.